pilot/pkg/proxy/envoy/v2/mem.go

```diff
@@ -235,6 +236,7 @@ func (sd *MemServiceDiscovery) SetEndpoints(service string, namespace string, en
 
 	}
 
+	sd.mutex.Unlock()
```

Comment:
ditto

Reply:
sorry, forgot to remove this. will remove